plugins/versionpress/src/Git/CommitMessage.php

```diff
@@ -57,6 +57,19 @@ public function getSubject()
     }
 
     /**
+     * Returns commit message subject without prefix set in constant VP_COMMIT_MESSAGE_PREFIX
+     * 
+     * @return string
+     */
+    public function getUnprefixedSubject()
```

Comment:
> because there will be no benefits of having it in commit messages

We don't use it anyway. It's just for users who use other tool than VersionPress to see the commits.

OK, we will change this later.